dragonflybsd/README.md

```diff
@@ -0,0 +1,137 @@
+# DragonflyBSD fwupd port notes
+
+## Preparation
+
+- Installed packages
+    - qemu-system-x86-64
+    - ovmf
+- Downloaded
+[latest version](https://mirror-master.dragonflybsd.org/iso-images/?C=M;O=D) of
+DragonflyBSD release in uncompressed `.iso` format
+- Created Virtual disk with `qemu-img` command:
+
+```
+$ qemu-img create -f qcow2 disk.qcow2 15G
+```
+
+> NOTE: HAMMER2 (default filesystem on DragonflyBSD) was designed to work with >
+50GB spaces. From 512MB to 1GB can be reserved for reblocking and UNDO/READ
+FIO. 15GB virtual disk was verified and should be enough for kernel development
+and verification purposes.
+
+## Scripts
+
+These scripts do not provide full automation but still can be useful
+
+- **vm/setup.sh** - run QEMU machine first time
```

Comment:
We could provide some information that this runs the DflyBSD installer and the user should follow some default installation steps (and that this is ran one time only)